### PR TITLE
[ubuntu-dev] Switch to Node 9.8.0 (current)

### PR DESCRIPTION
**Pull request by ishitatsuyuki**

### CODE REVIEW

**Comment by ishitatsuyuki** [comment]
This is not strictly needed now because the original intention was to use Node 9's http2 module but I end up with node-spdy, a npm package with broad compatibility.

Should we stay at LTS or migrate to current nonetheless?

**Comment by jankeromnes** [comment]
> Should we stay at LTS or migrate to current nonetheless?

We try to have all useful developer tools at their latest stable version, so I guess let's try one of your images with Node 9, and upgrade to Current if it works, or upgrade to the latest LTS if it doesn't.

File: ubuntu-dev/ubuntu-dev.dockerfile [hunk 1]
```diff
@@ -147,9 +147,9 @@ RUN wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.s
 # Install latest Node.js, npm and Yarn.
 ENV NVM_DIR="/home/user/.nvm"
 RUN . $NVM_DIR/nvm.sh \
- && nvm install v8.9.4 \
+ && nvm install v9.8.0 \
```
Comment (jankeromnes) [inline ubuntu-dev/ubuntu-dev.dockerfile]: Nit: Please upgrade to `v9.9.0` now. 😋

**Comment by jankeromnes** [comment]
Looks good to me, thanks a lot!

**Pull request merged**